app/overrides/spree/admin/shared/_configuration_menu.rb

```diff
@@ -1,7 +1,7 @@
 Deface::Override.new(
```

Comment:
These defaces can be replaced with modifications to `Spree::Config.menu_items` for greatly increased reliability.